service.watchedlist/addon.xml

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.watchedlist"
+    name="WatchedList"
+    version="1.3.0"
+    provider-name="schapplm">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.buggalo" version="1.2.0" />
+        <import addon="script.module.myconnpy" version="8.0.18" />
+        <!--<import addon="script.module.pydevd" version="3.4.1"/> --> <!-- Required for remote debugging in Linux -->
+        <import addon="script.module.dropbox" version="9.4.0"/> <!-- For Dropbox synchronisation -->
+        <import addon="script.module.dropbox_auth" version="1.0.0" optional="true"/> <!-- For Dropbox authentification -->
+    </requires>
+    <extension point="xbmc.python.script" library="manual.py"> <!--Python code for manual start in the addons menu-->
+        <provides>executable</provides> <!-- Addon is shown under Programs -->
+    </extension>
+    <extension point="xbmc.service" library="default.py" /> <!-- Python code for autostart -->
+    <extension point="xbmc.python.module" library="lib" /> <!-- entry to access the WatchedList from other addons -->
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Save watched state of movies and TV-series episodes independent from filenames</summary>
+        <summary lang="de_DE">Gesehen-Status von Filmen und Serien-Episoden unabhängig von Dateinamen speichern</summary>
+        <description lang="en_GB">
+        Export: Searches the Kodi-Database for watched files. Determine imdb-id and thetvdb-id to identify movies and TV-episodes. Then save the list to a new independent table.
+        Import: Set the watched state for each video file in Kodi.
+        Automatic background process without user interaction.</description>
+        <description lang="de_DE">
+        Export: Durchsucht die Kodi-Datenbank nach gesehenen Filmen und ordnet den Dateien die eindeutigen Kennnummern von imdb und TheTVDB zu. Gesehene Filme und Episoden werden jeweils in einer eigenen Datenbank gespeichert.
+        Import: Der Gesehen-Status von Medien in Kodi wird aus der Gesehenen-Tabelle geholt.
+        Die Synchronisierung geschieht automatisch im Hintergrund ohne Benutzereingriff.</description>
+        <disclaimer lang="en_GB">This script does not regard the file format. A Movie watched in DVD quality will also set    the 1080p version as watched.</disclaimer>
+        <disclaimer lang="de_DE">Dieses Script ignoriert das Dateiformat. Ein gesehener Film in DVD-Qualität wird auch in der BlueRay-Version als gesehen gekennzeichnet.</disclaimer>
+        <platform>all</platform>
+        <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+        <forum>https://forum.kodi.tv/showthread.php?tid=164652</forum>
+        <website>https://kodi.wiki/view/Add-on:WatchedList</website>
+        <email></email>
```

Comment:
drop if empty

Reply:
same, if email is empty just remove from addon.xml

Reply:
should be already removed in the latest commit